### ISSUE TITLE
Running a get request with randomly generated ids

### ISSUE THREAD

**Issue by eugenemiretsky** [elastic/rally#106]
Is it possible to do now? 
If not, it would be nice to be able to do that. 
The idea is just to test Get API performance with a random doc coverage.

**Comment by danielmitterdorfer** [comment]
tl;dr no, it's currently not possible.

I am currently working on the [0.4.0 release](https://github.com/elastic/rally/milestones/0.4.0) and will implement a new load generator in Rally which is more flexible. I already toyed around with the idea of allowing generators in the track specification but this will be the second step. My idea is that you can define parameters for each operation (search, bulk indexing, etc.). They can either be static or generated dynamically. You can then define a generator function that will be invoked by Rally each time the operation is invoked. I think this should allow you to achieve what you have in mind. I don't think that this will make it in 0.4.0 but I think it's reasonable to expect it in 0.5.0.

I've created now #107 and closing this one as it is more specific than what I have in mind. If you have further ideas or suggestions for this feature, please comment in #107. I'd really appreciate it. :)
